src/blocks/cpu.rs

```diff
@@ -134,8 +134,11 @@ impl Block for Cpu {
             _ => State::Idle,
         });
 
-        self.utilization.set_text(format!("{:02}%", utilization));
-
+        if utilization < 100  {// 2 digits, pad with extra space to ensure consistent length
+            self.utilization.set_text(format!("  {:02}%", utilization));
+        } else {
+            self.utilization.set_text(format!("{:02}%", utilization));
```

Comment:
Why not `"{:03}%"` instead? Wouldn't that do the same thing?

Reply:
Well, it will pad with a zero instead of spaces, but ````099%````, ````055%```` or ````011%```` looks a bit stupid imo.

Reply:
I'd prefer to have less duplication here, we can use an expression for the optional space instead of an surrounding if:
```Rust
format!(" {}{:02}%", if utilization < 100 {" "} else {""}, utilization)
```

Reply:
Nevermind. `format!("{:3}%", utilization)` is what we're looking for

Reply:
Oops, my bad. That's the one I meant.

Reply:
````{:3}```` makes jumping even worse here... :/ 
I guess a space does not have the same width as number in the default font?

Reply:
Then just use a Monospace font

Reply:
Well I do use ````      font pango:DejaVu Sans Mono, FontAwesome 12```` as suggested in the readme.md...
I wonder what is happening there.